dist/index.html

```diff
@@ -644,14 +644,14 @@ <h3 class="showcase-row__heading my-2">Manage Donations</h3>
       <div class="dots-bg w-64 h-48 ml-28"></div>
     </section>
 
-    <section class="news font-body bg-c800 mb-48 pt-18 pb-40 relative">
+    <section class="news font-body bg-c800 mb-48 pt-18 pb-1 md:pb-40 relative">
```

Comment:
It looks fine except the `margin-bottom` value as `mb-48` is applied in small screens too.
![news-section](https://user-images.githubusercontent.com/8650435/79895969-c6f13700-8418-11ea-87c6-f2be06502311.PNG)